src/renderers/dom/client/ReactReconcileTransaction.js

```diff
@@ -7,6 +7,7 @@
  * of patent rights can be found in the PATENTS file in the same directory.
  *
  * @providesModule ReactReconcileTransaction
+ * @flow weak
```

Comment:
Oh! maybe you don't want those! Can delete them if you ask, but flow helped me though (once again!)...